### PR TITLE
JavaScript capstone project  API-based webapp

### PR DESCRIPTION
**Pull request by Bria222**
> created webpack application 
> Used TVmaze API to fetch movies data
> implemented likes functionality from an API
> wrote tests to test for counter function and comments
> implemented  ES6 syntax

### CODE REVIEW

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Bria222 & @ARANY & @V0pe, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by mmsesay** [comment]
You can also consider:

- [ ] You did well pitching your project in the video and also showing some parts of the code which is great. However, I will suggest you get other members of your team to also present some features of your project. I didn't see any of them in the video. Please reach out to them and do the video together.

- [ ] I noticed three contributors to this project. Was your group made up of 3 students?  If that's is true, you were supposed to work on the reservations pop part as well.

**Comment by Bria222** [comment]
@mmsesay  we are two my partner has two GitHub accounts that's why it's showing three members  but we are only two

**Comment by mmsesay** [comment]
Good to know @Bria222. Please work on the other change.

**Comment by Bria222** [comment]
@mmsesay the video right

**Comment by V0pe** [comment]
@mmsesay Just like Brian say, it is only two...unfortunately, I have one separate GitHub account linked to the local repo which is different from my remote repo. I have tried solving this, but the issue persist. Apologise

**Comment by Bria222** [comment]
@mmsesay thanks for the review we have fixed the issue and recorded a video

File: README.md [hunk 1]
```diff
@@ -23,12 +18,18 @@ To get a local copy up and running follow these simple steps:
 2. Press the "Code" button and copy the link.
 3. Clone it using git command `git clone <https://github.com/Bria222/KanbanBoard.github.io>`.
 
+## to test the application
+> npm install --save-dev jest
+> npm test
+
```
Comment (CollinsTatang) [inline README.md:24]: - [ ] Please structure your readme file following the [Microverse](https://github.com/microverseinc/readme-template)
- [ ] Please deploy your project and add the live demo link to your readme file

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Bria222 , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Bria222) [reply README.md:24]: @CollinsTatang  okay thanks let me do so

Comment (CollinsTatang) [reply README.md:24]: Ok @Bria222

Comment (Bria222) [reply README.md]: @CollinsTatang  i have done the changes can you please check

Comment (CollinsTatang) [reply README.md]: @Bria222  you still have linters errors, please fix it before you submit for review

**Comment by Bria222** [comment]
@CollinsTatang thanks for the feedback I have implemented the requested changes

File: README.md (hunk 1 above)
Comment (V0pe) [reply README.md]: @CollinsTatang We have checked the linters. It is okay.

**Review by Deepakdanger — APPROVED** [review]
**Hi @Team**, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Bria222, and @V0pe👋, 

### You did a marvelous job on the group capstone project and this is one of the best projects I have seen so far!

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- Good commit messages. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**